### PR TITLE
feat(plugin) FakeNitro: Allow customising hyperlink text

### PR DESCRIPTION
**Pull request by LualtOfficial**
If you use U+2800 then links will essentially be invisible which imo looks way cleaner. Image below is how it should look with the fix.

ksnip_20240218-163048

### CODE REVIEW

**Comment by EdVraz** [comment]
This is a thing of preference. I'd prefer to see the emoji name as link rather than nothing. 

I would suggest making the text of the hyperlink configurable instead, with the default being the emoji name and the user being able to change it to anything.

File: src/plugins/fakeNitro/index.ts [hunk 1]
```diff
@@ -162,6 +162,11 @@ const settings = definePluginSettings({
         description: "Whether to use hyperlinks when sending fake emojis and stickers",
         type: OptionType.BOOLEAN,
         default: true
+    },
+    hyperLinkText: {
+        description: "What text the hyperlink should use, if empty it uses the emoji name",
```
Comment (Vendicated) [inline src/plugins/fakeNitro/index.ts:167]: i don't like the empty approach. instead, support a placeholder of `{{NAME}}`

so
```
:{{NAME}}: -> :foo:
{{NAME}} -> foo
```

then the default would be just `{{NAME}}`

**Comment by Vendicated** [comment]
please do not pull request from the main branch. use features branches

**Comment by Vendicated** [comment]
thanks!

**Pull request merged**